apps/arena/lib/arena/game_updater.ex

```diff
@@ -491,6 +495,8 @@ defmodule Arena.GameUpdater do
   end
 
   defp report_game_results(state, winner_id) do
+    match_id = Ecto.UUID.generate()
```

Comment:
You can remove this variable and just call generate/0 inside the map.

Reply:
detroyed